repositories/OptionRepository.js

```diff
@@ -32,16 +32,19 @@ module.exports.findAll = function findAll(
     .then(map(fromDb));
 };
 
-module.exports.findOne = function findOne(
-  where = {},
-  orderBy = "created_at",
-  direction = "asc"
+module.exports.findExclusiveOptionByAnswerId = function findExclusiveOptionByAnswerId(
```

Comment:
I still think this is unnecessary as we can just utilise `findAll()`

Reply:
I disagree here, I'd rather have a specialised function for something like this, as I'd like to make it very obvious what it's doing when we're calling it from the resolvers file, plus it just allows us to keep the resolver functions dumber.

Reply:
I understand your point of view.  However, I would prefer not to pollute the repositories with lots of wrapper functions. 

Nevertheless, I'm happy to resolve this conversation and we can discuss in the future if it becomes problematic.